templates/java/org/prism/Nodes.java.erb

```diff
@@ -175,8 +175,8 @@ public abstract class Nodes {
     <%- nodes.each do |node| -%>
 
     /**
-    <%- node.comment.each_line do |line| -%>
-     *<%= line.prepend(" ").rstrip %>
+    <%- node.each_comment_line do |line| -%>
+     * <%= line %>
```

Comment:
This approach will leave trailing spaces for empty lines in comments.
`*<%= line %>` + `yield line.prepend(" ").rstrip` would avoid that

Reply:
Ah I was wondering why that was there. I'll fix this.